doc/build/dts/intro-syntax-structure.rst

```diff
@@ -67,6 +69,12 @@ Devicetree nodes can also have *properties*. Properties are name/value pairs.
 Property values can be any sequence of bytes. In some cases, the values are an
 array of what are called *cells*. A cell is just a 32-bit unsigned integer.
 
+.. _dt-property-labels:
+
+Just like nodes, properties can also have *property labels*, which use the same
+syntax and share the same purpose as node labels (but for properties),
+described in the previous section.
+
```

Comment:
I believe this needs to be moved after following paragraph. Would also be useful to have a `bar` property showing an example of how this looks like.

Reply:
+1 to adding an example - I wasn't aware that there are *property labels*.  Is there any scoping limits to using a *property label*?